app/views/projects/index.html.erb

```diff
@@ -55,6 +55,18 @@
   </div>
 <% end %>
 
+<br/>
```

Comment:
Can you post a snapshot of this looks like? It should on the same line next to search.

Reply:
![image](https://user-images.githubusercontent.com/496330/77400325-fb011a00-6dd0-11ea-99b2-129f1c0c44a3.png)

![image](https://user-images.githubusercontent.com/496330/77400358-06ecdc00-6dd1-11ea-9f70-459a069df1e5.png)